### PR TITLE
[Quest] Let Sleeping Dogs Lie

### PR DESCRIPTION
**Pull request by jamesbradleym**
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my PR will be ignored.
- [x] I understand I should leave resolving conversations to the LandSandBoat team so that reviewers won't miss what was said.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have _**tested my code and the things my code has changed**_ since the last commit in the PR and will test after any later commits.

## What does this pull request do?

Implements the Windy quest: Let Sleeping Dogs Lie

Based on: https://github.com/AirSkyBoat/AirSkyBoat/pull/2565

Modifications to original:

Reformatted for LSB

Adds additional dialog for students after waking Paku-Nakku
Removes Obtained Item text when harvesting
Fixes item parameter dialogue for Pechiru-Mashiru
Adds additional dialog for Paku-Nakku after waking
Adds pre-quest text for Mashuu-Ajuu

[Capture](https://www.youtube.com/watch?v=GpnY5LCkQrA&feature=youtu.be)
[Capture](https://www.dropbox.com/scl/fi/8aj1zagl47w9tg5ue0jnl/Quest-Windurst-Let-Sleeping-Dogs-Lie.zip?rlkey=u8p2a0395xsoyo96gkhhbezjn&dl=0)

## Steps to test these changes

Run through quest Let Sleeping Dogs Lie

### CODE REVIEW

File: scripts/zones/Pashhow_Marshlands/DefaultActions.lua [hunk 1]
```diff
@@ -3,4 +3,5 @@ local ID = zones[xi.zone.PASHHOW_MARSHLANDS]
 return {
     ['Odyssean_Passage'] = { messageSpecial = ID.text.NOTHING_HAPPENS },
     ['Outpost_Gate']     = { messageSpecial = ID.text.GATE_IS_LOCKED },
+    ['QM1']              = { messageSpecial = ID.text.NOTHING_OUT_OF_ORDINARY },
```
Comment (Xaver-DaRed) [inline scripts/zones/Pashhow_Marshlands/DefaultActions.lua]: `QM1`?
qm1
```
INSERT INTO `npc_list` VALUES (17146515,'qm1','???',0,-105.147,24.999,-255.986,1,40,40,0,0,0,0,3,0x0000340000000000000000000000000000000000,0,'WOTG',0);
```

Comment (jamesbradleym) [reply scripts/zones/Pashhow_Marshlands/DefaultActions.lua]: That's why they pay you the big bucks 🫠
Fixed!

Comment (Xaver-DaRed) [reply scripts/zones/Pashhow_Marshlands/DefaultActions.lua]: If I got paid this wouldnt be a hobby

File: scripts/quests/windurst/Let_Sleeping_Dogs_Lie.lua [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+            },
+        },
+
+        [xi.zone.PORT_WINDURST] =
+        {
+            ['Maabu-Sonbu'] =
+            {
+                onTrigger = function(player, npc)
+                    if quest:getVar(player, 'Prog') == 0 then
+                        return quest:progressEvent(319, 653392653, 4155, 1102, 1020) -- 0, 0, 0, 0)
```
Comment (claywar) [inline scripts/quests/windurst/Let_Sleeping_Dogs_Lie.lua]: Can you please remove these commented values? Known that default is 0 for all params not specified.  Also, can you take a look into 4155, 1102, and 1020, as they smell like either keyitems or items.

This applies to all of the event parameters below as well.

Comment (jamesbradleym) [reply scripts/quests/windurst/Let_Sleeping_Dogs_Lie.lua]: Swapped in item vars! Thanks!

**Comment by jamesbradleym** [comment]
@Xaver-DaRed @claywar Anything else needed here? 😅

**Comment by jamesbradleym** [comment]
Closing this PR to reduce open PRs and prevent staleness—will revisit.

**Pull request closed without merging**